paasta_tools/api/api.py

```diff
@@ -254,7 +254,7 @@ def main(argv=None):
         os.path.join(sys.exec_prefix, "bin", "gunicorn"),
         "gunicorn",
         "-w",
-        "4",
+        str(load_system_paasta_config().get_paasta_api_number_workers()),
```

Comment:
i'll see if i can get puppet to restart the systemd service whenever the config file that this is in gets updated - otherwise, we'd need to manually do that whenever we want to change this value

Reply:
Oh I forgot we pass args to the api (like port, timeout) anyway, so if we made this an arg, we could control that in puppet anyway and maybe make the reloading a little less complicated.

But this works too

Reply:
oh, true...

that might be a better way to do things!